microprofile/fault-tolerance-smallrye/extension/src/main/java/org/wildfly/extension/microprofile/faulttolerance/MicroProfileFaultToleranceLogger.java

```diff
@@ -43,8 +42,8 @@
     @Message(id = 1, value = "Activating MicroProfile Fault Tolerance subsystem.")
     void activatingSubsystem();
 
-    @LogMessage(level = WARN)
-    @Message(id = 2, value = "Hystrix was already configured! Skipping configuration from deployment '%s'.")
-    void hystrixAlreadyConfigured(String deploymentName);
+//    @LogMessage(level = WARN)
+//    @Message(id = 2, value = "Hystrix was already configured! Skipping configuration from deployment '%s'.")
+//    void hystrixAlreadyConfigured(String deploymentName);
```

Comment:
Can this be removed completely?

Reply:
It can be removed, but best practice is to keep those in commented out so that messages do not get *accidentally* reused.

Reply:
Even if this message has never been part of a WildFly.Final version? It's just in WF19.Beta1, right?

Reply:
That's a good point – we don't have to be overly rigid here – since it was never in any Final version and this willl be backported. Removed now.